### ISSUE TITLE
Finalizing cut and names

### ISSUE THREAD

**Issue by bryates** [TopEFT/topcoffea#25]
We talked about this briefly last Friday (5/7), but we need to make sure all the cuts and their corresponding names, e.g. https://github.com/TopEFT/topcoffea/blob/05655b196484c08e20884678df3e5beab6312fb3/analysis/topEFT/topeft.py#L311-L328 and https://github.com/TopEFT/topcoffea/blob/05655b196484c08e20884678df3e5beab6312fb3/analysis/topEFT/topeft.py#L330-L335. We need to make sure all the channels are included (4l is missing) and all the jet/b-jet categories are accounted for.
@GonzalezFJR you said you might try to take a look, but let me know if you're too busy. I can always try updating these.

**Comment by GonzalezFJR** [comment]
I included the 4l categories and categorization according to the sign of the sum of the charge (following TOP-19-001). I also included 2D histograms for njet-nbtag categories, which should be enough to obtain any categorization that we need. 
This is currently in the addingSF branch, in which we're also adding the SFs and systematic categories. We'll open a new PR when some of the ongoing work is finished, probably befor tomorrow's meeting.
